src/test/java/hudson/tasks/AntTest.java

```diff
@@ -200,7 +200,8 @@ public void testParameterExpansion() throws Exception {
             assertTrue("Missing $BUILD_TAG: " + log, log.contains("vTAG=jenkins-test project-AX\\=is-1"));
             assertTrue("Missing $EXECUTOR_NUMBER: " + log, log.matches("(?s).*vEXEC=\\d.*"));
             // $NODE_NAME is expected to be empty when running on master.. not checking.
-            assertTrue("Missing $NODE_LABELS: " + log, log.contains("vLAB=master"));
+            String label = r.jenkins.getVersion().isOlderThan(new VersionNumber("2.307")) ? "master" : "built-in";
```

Comment:
Why not just amend this to look up the self-label for `r.jenkins`, so that you do not need to hard-code a value? Basically the opposite of https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/167#discussion_r703681894

Reply:
resolved in 6e2335f